src/main/java/nl/tudelft/contextproject/Main.java

```diff
@@ -50,6 +50,7 @@
 	public static final float RESOLUTION = 1.0f;
 	//If the mirror window is shown
 	public static final boolean MIRROR_WINDOW = true;
+	public static final Float TIME_LIMIT = new Float(300);

```

Comment:
Use `300f` instead of `new Float(300)`; (Never use new for primitive wrapper objects. If need be, use `Float.valueOf()`)